rmf_fleet_adapter/rmf_rxcpp/test/test_Transport.cpp

```diff
@@ -44,42 +44,39 @@ TEST_CASE("publish subscribe loopback", "[Transport]")
   auto obs =
     transport->create_observable<std_msgs::msg::String>(topic_name, 10);
 
-  SECTION("can receive subscription")
-  {
-    std_msgs::msg::String msg{};
-    msg.data = "hello";
-    auto timer =
-      transport->create_wall_timer(std::chrono::milliseconds(100),
-        [&publisher, &msg]()
-        {
-          publisher->publish(msg);
-        });
+//  SECTION("can receive subscription")
```

Comment:
Ah yeah, they were commented out because I needed to change the API that these tests depend on. I should replace these with new tests that are relevant to the new API.

Reply:
Fixed: 6391ed3